packages/react/package.json

```diff
@@ -166,6 +166,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.13.16",
+    "@babel/helper-builder-react-jsx": "^7.12.13",
```

Comment:
Any idea why this was needed? It doesn't come up in a yarn why. Just curious what's causing the need for it now.

Reply:
I'm not sure, I'm getting build failures with a failure to resolve that package